crates/transaction-fuzzer/src/account_universe/transfer_gen.rs

```diff
@@ -422,6 +422,11 @@ impl AUTransactionGen for P2PTransferGenRandomGasRandomPriceRandomSponsorship {
         // *sender.current_balances.last().unwrap();
         let rgp = exec.get_reference_gas_price();
         let run_info = RunInfo::new(gas_balance, rgp, self);
+        let reference_gas_price = if PROTOCOL_CONFIG.fixed_base_fee() {
+            PROTOCOL_CONFIG.base_gas_price()
+        } else {
+            exec.get_reference_gas_price()
+        };
```

Comment:
This code should be extracted in a function and used where it is needed because it is already duplicated in the authority per epoch store.